Makefile

```diff
@@ -1,7 +1,7 @@
-KUTTL_VERSION=0.5.0
+KUTTL_VERSION=0.6.1
```

Comment:
Why are you bumping kuttl?

Reply:
That's the version that the current build config in TeamCity uses, just making sure Dispatch one is up to date too.